index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <!-- navigation bar -->
+    <nav class="navbar">
+        <div class="logo">MyBlogs</div>
+       
+        <ul class="nav-links">
+            <li><a href="#">Blogs</a></li>
+            <li><a href="#">Contacts</a></li>
+            <li><a href="#">About</a></li>
+        </ul>
+    </nav>
+    
+
+    <!-- content container where posts will be shown  -->
+    <div class="content-container">
+         <!-- dynamic data of posts will load here  -->
+    </div>
+
+    <!-- pagination container  -->
+    <span class="pagination-box">
+       <button class="prev">Prev</button>
+       <span class="paginated-btn-grp">
+           
+       </span>
+       <button class="next">Next</button>
+    </span>
+    
+    <script src="script.js" ></script> 
```

Comment:
do not messup everything in the one file.

Reply:
Done, made class based structure and modularized whole code.